### DISCUSSION TITLE
Posted In An "Issues" Section, Received GitHub Email With A Virus?

### DISCUSSION THREAD

**Discussion by GitAlexToTheGreek** [community/community#136990] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi there,

While waiting for feedback from GitHub Support, I'm wondering if anyone has insight as to how the following could happen? (The "View it on GitHub" link was valid, pointing to the real thread, with the sender's comment removed.)

https://github.com/sabrogden/Ditto/issues/707#issuecomment-2311427590

> Heads up @sabrogden
> 
> Wondering if you're able to view who the "and others" is referring to...
> 
> ![Screenshot_20240826_184846_Chrome](https://private-user-images.githubusercontent.com/59787873/361624581-a1dc827d-899f-4001-a37f-ec2e26e2dd0f.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ3OTcwOTQsIm5iZiI6MTcyNDc5Njc5NCwicGF0aCI6Ii81OTc4Nzg3My8zNjE2MjQ1ODEtYTFkYzgyN2QtODk5Zi00MDAxLWEzN2YtZWMyZTI2ZTJkZDBmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI3VDIyMTMxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFlNjIwMzljN2U0M2E0MjMwNzhjODQyYWZmMDYxYmM5MjcwNzJmNTY1ZGVlN2MwYWNlZGM2ZmM3MGY1NDk2ZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Datmc6Vhp8SwkPNMxQ8xECs8GkMoHzlrOzcjpJXdLqw)
> 
> ...because I received an email not long after making the thread, from someone posing (?) as a dev, containing a link to a virus (not affected because used an online sandbox to view the contents)...
> 
> ![Screenshot_20240825_224704_Gmail](https://private-user-images.githubusercontent.com/59787873/361626724-803b7db9-108a-45a2-8fb1-f54b950e03fb.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ3OTcwOTQsIm5iZiI6MTcyNDc5Njc5NCwicGF0aCI6Ii81OTc4Nzg3My8zNjE2MjY3MjQtODAzYjdkYjktMTA4YS00NWEyLThmYjEtZjU0Yjk1MGUwM2ZiLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI3VDIyMTMxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZkYzEzMDE3M2QyZGFjOTk3YjI5M2IyMjg5NTU1Mzk4MDVmZTFmNjBhYmNlOTQwMTUyNjJjZDBhZGJmNTYxMjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gVYpoCAqLexOki2sZVXWf-il7paqaIAFFCMn_Zc13Eg)
> 
> Do you recognize the sender's name as an actual member of this repository?
> 
> I couldn't locate the "report spam/abuse" button for this "Issues" section so adding it here...
> 
> Wondering what you make of all this? 😬

**Comment by DecimalTurn** [comment, marked as answer]
There is a lot of discussions about this problem here: https://github.com/orgs/community/discussions/136836

The account that posted the malicious link was probably a bot or a compromised account. GitHub has been taking measures to curb this malware campaign and this is why most of them have been deleted.

**Reply by GitAlexToTheGreek** [reply]
> There is a lot of discussions about this problem here: https://github.com/orgs/community/discussions/136836
> 
> The account that posted the malicious link was probably a bot or a compromised account. GitHub has been taking measures to curb this malware campaign and this is why most of them have been deleted.

Thanks for clarifying, Martin!

**Reply by DecimalTurn** [reply]
> are you thinking this [is] why [they are] suspending accounts?

Yes
